adapters/oidc/spring-boot/src/main/java/org/keycloak/adapters/springboot/KeycloakSpringBootConfiguration.java

```diff
@@ -209,7 +209,8 @@ public void customize(Server server) {
                 }
             }
 
-            WebAppContext webAppContext = server.getBean(WebAppContext.class);
+            WebAppContext webAppContext = (WebAppContext) server.getHandler();;
```

Comment:
For sure, we need to document it but even here we could maybe make it a bit more solid by first checking is there is a `WebContext` bean and if not fallback to `.getHandler()` , not sure it covers all the cases but maybe better than just relying on `.getHandler()`.
Let me update this.